target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -58,6 +58,7 @@ get_status_led() {
 		status_led="$board:blue:wlan"
 		;;
 	all0239-3g|\
+	ac1200pro|\
```

Comment:
Ah sorry, missed that the with the last review.

Please keep alphabetical order in that block. The ac1200pro has to be before the ALL0239-3g. Afterwards move the whole block up so that the alphabetical order in the file is kept.